dev-libs/libprelude/libprelude-5.1.1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python{2_7,3_{5,6,7}} )
+USE_RUBY="ruby22 ruby23 ruby24 ruby25"
+DISTUTILS_OPTIONAL=1
+
+inherit autotools distutils-r1 ruby-single
+
+DESCRIPTION="Prelude-SIEM Framework Library"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/5.1.0/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="lua perl python ruby"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="net-libs/gnutls:=
+	lua? ( dev-lang/lua:* )
+	perl? ( dev-lang/perl:= virtual/perl-ExtUtils-MakeMaker )
+	python? ( ${PYTHON_DEPS} )
+	ruby? ( ${RUBY_DEPS} )"
```

Comment:
These are BDEPEND in EAPI-7.

Reply:
I'm not sure if these are BDEPEND. They are for language bindings, no? If you have the ruby use flag, it will build ruby bindings for libprelude. They should be in RDEPEND.

If these interpreters are also executed on the host during the build process, they should also be listed in BDEPEND.

Reply:
I thought it initializes the interpr just like it does with python, ie chooses one compatible ruby binary. It was just a feeling though.